docs/production/mobile-push-notifications.md

```diff
@@ -18,25 +18,26 @@ follows:
    outgoing HTTP proxy](deployment.md#customizing-the-outgoing-http-proxy)
    first.
 
+   1. Uncomment the
+      `ZULIP_SERVICES = {"mobile_push", "submit_usage_statistics"}` line
+      in your `/etc/zulip/settings.py` file (i.e., remove the `#` at the
+      start of the line), and [restart your Zulip
+      server](settings.md#making-changes).
```

Comment:
Hmm, we probably need some discussion for servers with old versions or legacy comments. Maybe just "If you're running Zulip 9.0 or greater, .... Otherwise, see these instructions" with "these instructions" being a link to the /8.4/ version of this page on ReadTheDocs?

Reply:
Hmm yeah, might be good as a visible `Note` or something. Will be good to keep in mind when writing the docs on this once we commit to this current new way of configuring.

Reply:
I moved the "restart the server" part further down, since you should do it after you decide if you're sending usage statistics, ideally, just because you're already in the right file.